docs/tutorial/part-five/index.md

```diff
@@ -75,42 +75,42 @@ module.exports = {
 }
 ```
 
-Save that and restart the gatsby development server. Then open up GraphiQL again.
+सहेजें और gatsby विकास सर्वर को पुनरारंभ करें। इसके बाद फिर से GraphiQL को खोलें।
 
-In the explorer pane, you'll see `allFile` and `file` available as selections:
+एक्सप्लोरर फलक में, आप `allFile` और` file` चयनों के रूप में उपलब्ध देखेंगे:
```

Comment:
`चयनों` => `सिलेक्शनस`

Reply:
`सिलेक्शनस` => `सलेक्शनस`
Now that I am reading this, `सलेक्शनस` sounds the right word. Can you update this.
Sorry about the previous feedback